test/tools/libtesteth/TestHelper.cpp

```diff
@@ -534,6 +534,36 @@ void executeTests(const string& _name, const string& _testPathAppendix, const st
 	}
 }
 
+//A general post parse function which converts filler file into final json test.
+void postParseJson(json_spirit::mValue& _obj, string const& _testfilename)
+{
+	if (_obj.type() == json_spirit::obj_type)
+	{
+		std::list<string> removeList;
+		for (auto& i: _obj.get_obj())
+		{
+			if (i.first.substr(0, 2) == "//")
+			{
+				removeList.push_back(i.first);
+				continue;
+			}
+
+			postParseJson(i.second);
+		}
+		for (auto& i: removeList)
+			_obj.get_obj().erase(_obj.get_obj().find(i));
+	}
+	else if (_obj.type() == json_spirit::array_type)
+	{
+		for (auto& i: _obj.get_array())
+			postParseJson(i);
+	}
+
+	//Add client info to the top level of the test
+	if (!_testfilename.empty())
```

Comment:
That's wicked.  Maybe put it back out of this function.  I think that keeps this function a simple syntactic transformation, not adding information, just removing comments.